mautic/mautic
# Feature/add reply event in campaign

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR will check any reply to an email as a campaign action.

This will help to create campaigns based on reply to an email. It will fetch the email replied using in-reply-with header and check for the source message and update stats of email.

#### Steps to test this PR:
1. Apply this PR
2. execute app/console doctrine:migrations:migrate
3. Clear the cache
4. It will show option 'Reply' in Monitored Inbox Settings in configuration to configure
5. Create a new campaign with  segment and "Send email" and "Reply To" condition. 
6. Execute the app/console mautic:campaigns:update and app/console mautic:campaigns:trigger commands.
7. Check the reply condition is works.
